android/build.gradle

```diff
@@ -20,17 +35,22 @@ android {
 }
 
 repositories {
+    mavenLocal()
     maven {
         // All of React Native (JS, Obj-C sources, Android binaries) is installed from npm
-        // Matches the RN Hello World template
-        // https://github.com/facebook/react-native/blob/1e8f3b11027fe0a7514b4fc97d0798d3c64bc895/local-cli/templates/HelloWorld/android/build.gradle#L21
-        url "$projectDir/../node_modules/react-native/android"
+        url "$rootDir/../node_modules/react-native/android"
+    }
+    maven {
+        // Android JSC is installed from npm
+        url "$rootDir/../node_modules/jsc-android/dist"
     }
-    mavenCentral()
```

Comment:
I’m not super familiar with these configs, but are you sure it won’t affect anything?

Reply:
It will not affect anything. `mavenCentral` is the "old" entry which has been replaced by `jcenter` (which is a superset of `mavenCentral`).